web/app/view/BaseDialog.js

```diff
@@ -22,6 +22,7 @@ Ext.define('Traccar.view.BaseDialog', {
     resizable: false,
     modal: true,
     autoScroll: true,
+    constrain: true,
```

Comment:
Do not allow dialog windows be moved or expanded outside parent container (`document.body` by default)